lib/proxy/ftp/ctrl.c

```diff
@@ -498,7 +500,7 @@ int proxy_ftp_ctrl_send_resp(pool *p, conn_t *ctrl_conn, pr_response_t *resp,
   }
 
   if (resp_nlines > 1) {
-    pr_response_send_raw("%s%s", resp->num, resp->msg);
+    pr_response_send_raw("%s-%s", resp->num, resp->msg);
```

Comment:
This is the key change, as we were not properly sending the `-` after the response code, which is what indicates a properly formatted multi-line FTP response.